Structure_Engine/Query/Visualize.cs

```diff
@@ -54,6 +52,9 @@ public static partial class Query
         [Output("lines", "A list of lines representing the load.")]
         public static List<ICurve> Visualize(this AreaUniformTemperatureLoad areaTempLoad, double scaleFactor = 1.0, bool displayForces = true, bool displayMoments = true, bool asResultants = true, bool edgeDisplay = true, bool gridDisplay = false)
         {
+            if (areaTempLoad.IsNull())
+                return null;
```

Comment:
Some scope to just return an empty list here instead of nulls. Same for all methods in this file